apps/web/src/pages/Applications/ApplicationSelfDeclarationPage/SelfDeclaration/CommunitySelection.tsx

```diff
@@ -73,7 +95,13 @@ export const CommunityList = ({ labels }: CommunityListProps) => {
                 }),
               },
             ]}
+            aria-describedby={customAlertId}
           />
+          {formState.errors.firstNationsCustom && (
+            <Field.Error id={customAlertId}>
```

Comment:
Could we add some margin here? 

```suggestion
            <Field.Error id={customAlertId} data-h2-margin-top="base(x.25)">
```